examples/v2/chat2.nim

```diff
@@ -292,6 +294,12 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
     
     node.wakuStore.setPeer(parsePeerInfo(storenode))
 
+    if conf.swap:
+      var report =  node.wakuSwap.balance()
+      echo "Total Credit: ", report.totalCredit
```

Comment:
Agree with @oskarth re aggregating in a report vs just printing the current accounting balances against each peer (we want the latter, and it's much simpler). Also, is this `echo` just temporary? We probably want to print to a log (say `info` level or `debug` level) or set a metric. `echo` will just write to `stdout`.

Reply:
For some reasons, there is an issue with building when `info` or `debug` is used. Says `Log statements should use keyword parameters to specify the log event properties`

Reply:
It means that there is something unexpected (likely in your syntax) in the properties you assigned to log. In other words, for the above your format should be something like:
```
info "Total credit", totalCredit=report.totalCredit
```

Reply:
I currently updated the script, so now the implementation is :

```
for peerId, balance in node.wakuSwap.accounting:
  info "Balance for ", peerId, ": ", balance
```

But it still has the same issue. However, replacing info with echo works

Reply:
Ah, yes, that won't work. Note that the log is not a string formatter but a way to present _properties_ with a log message. See https://github.com/status-im/nim-chronicles#introduction

The syntax is:
```
<loglevel> "messageString", <property1Name>=<property1Value>, <property2Name>=<property2Value>, ...
```

In other words, you might do something like
```
info "Balance for peer", peerId=peerId, balance=balance 
```

Reply:
Ohh Thanks. This was really helpful. So just to be sure, `info`, `debug`, and similar commands in him-chronicles only stores the logs in the log file, i.e `chat2.log`. It does not necessarily print out to the console as well yea?

Reply:
It will log to whatever you've set your log destination, which could be a file. In your case you haven't explicitly set a log destination, so it will log to stdout (most likely the console) by default. See https://github.com/status-im/nim-chronicles#chronicles_default_output_device

For now, I still see some `echo` commands in your code, which should not be used other than in applications specifically designed to interact with `stdout`. This may be something you're planning to address, so this is just an FYI. If some log serves only a debugging purpose, I'll use `debug` or `trace` level. `trace` logs are disabled by default, so is appropriate for very high-rate logs (e.g. if you're logging something for every message received).

Reply:
Ah, sorry, reread your comment and realised that you're using `chat2` and not just a plain `wakunode2`. 😄  `chat2` indeed logs to a file rather than `stdout`. I'd suggest that you use `wakunode2` for tests though. While `chat2` is convenient, it may not be the best tool for general development - the suppressed logs being one good reason.